images/win/scripts/Installers/Install-Git.ps1

```diff
@@ -37,7 +37,7 @@ Choco-Install -PackageName hub
 [Environment]::SetEnvironmentVariable("GCM_INTERACTIVE", "Never", [System.EnvironmentVariableTarget]::Machine)
 
 if (Test-IsWin16) {
-    $env:Path += ";$env:ProgramFiles\OpenSSH-Win64"
+    $env:Path += ";$env:ProgramFiles\Git\usr\bin\"
```

Comment:
This is a Git for Windows internal tool, isn't it?

Reply:
@eine, The `ssh-keyscan.exe`  is not available by default on Windows Server 2016. We decided to use `ssh-keyscan.exe` ,only during installation phrase, from git for windows as long as the util will not be deleted.

Reply:
> The `ssh-keyscan.exe` is not available by default on Windows Server 2016. We decided to use `ssh-keyscan.exe` ,only during installation phrase, from git for windows as long as the util will not be deleted.

@al-cheb, I understand that, but adding `Git\usr\bin\` defeats the purpose of this issue. `ssh-keyscan.exe` is available in MSYS2's `/usr/bin` too:

```sh
eine@DESKTOP MINGW64 /t/
# which ssh-keyscan
/usr/bin/ssh-keyscan
```

Reply:
Thank you. Fixed.